CHANGES.rst

```diff
@@ -196,6 +196,7 @@ New Features
 
   - Added furlong to imperial units. [#3529]
   - Added mil to imperial units. [#3716]
+  - Added stone to imperial units.
```

Comment:
Indeed. This was the very reason I didn't included the changelog file in the first commit :)